setup.cfg

```diff
@@ -1,6 +0,0 @@
-[flake8]
-max-line-length = 88
```

Comment:
Can we keep a flake8 line length config in the interim?

Reply:
Do you want to keep running flake8?

ruff defaults to 88 line length in our E checks:
https://docs.astral.sh/ruff/rules/line-too-long/
https://docs.astral.sh/ruff/settings/#line-length

Reply:
Yes, I haven't switched to `ruff` locally, and am not sure when I will. I'm sure I will eventually, but no time right now.

Reply:
:( It doesn't seem ideal to keep that around when we don't include flake8 in any of the installation/documentation instructions or checks. But, I did add the file back now.